less/home-page.less

```diff
@@ -19,12 +30,25 @@
     flex-wrap: wrap;
     width: 100%;
 
+    @media @phone {
+      width: 100%;
+      padding: 5;
+    }
+
     img {
       width: 28%;
       margin: 20px 2.5%;
       border-radius: 20px;
       height: 100%;
     }
+
+    @media @phone {
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      padding: 0;
+
+    }
   }
 
```

Comment:
Candidate for a // Space Images comment here, IMO! It breaks up the code and keeps it easy to read at a glance.